src/Commands.hs

```diff
@@ -841,3 +842,13 @@ toSymbols x = x
 
 commandHash :: UnaryCommandCallback
 commandHash ctx v = pure (ctx, Right (XObj (Num LongTy (Integral (hash v))) (Just dummyInfo) (Just LongTy)))
+
+commandParse :: UnaryCommandCallback
+commandParse ctx (XObj (Str s) i _) =
+  case parse s "command:parse" of
```

Comment:
`pure $ case ...` so it's easier to notice that all cases are pure?

Reply:
Thanks, I updated this portion!